docs/content.zh/docs/connectors/datastream/kafka.md

```diff
@@ -391,7 +391,8 @@ stream.addSink(myProducer)
 
 除此之外，我们还可以使用下列配置方法：
 
-- {{< javadoc name="setWriteTimestampToKafka(boolean writeTimestampToKafka)" file="horg/apache/flink/streaming/connectors/kafka/FlinkKafkaProducer.html#setWriteTimestampToKafka-boolean-" >}}，给每条记录设置时间戳
+- {{< javadoc name="set
```

Comment:
This new line is failing the azure build unfortunately (and we need green build to be allowed to merge a PR). Could you fix it and squash already all commits?

Reply:
Sure. Let me fix it.